src/JWT/Builder/JwtBuilder.cs

```diff
@@ -268,19 +281,26 @@ private void TryCreateValidator()
         private void EnsureCanBuild()
         {
             if (!CanBuild())
-                throw new InvalidOperationException("Can't build a token. Check if you have call all of the followng methods:\r\n" +
-                                                    $"-{nameof(WithAlgorithm)}\r\n" +
-                                                    $"-{nameof(WithSerializer)}\r\n" +
-                                                    $"-{nameof(WithUrlEncoder)}.");
+                throw new InvalidOperationException(
+                    "Can't build a token. Check if you have call all of the followng methods:\r\n" +
+                    $"-{nameof(WithAlgorithm)}\r\n" +
+                    $"-{nameof(WithSerializer)}\r\n" +
+                    $"-{nameof(WithUrlEncoder)}.");
+            
+            if(!HasOnlyOneSecret())
```

Comment:
Please add space after `if`. It looks like you have unusual VS or Resharper settings :)